tests/utils/profilehandler/profile_handler.go

```diff
@@ -565,3 +566,153 @@ func CreateClusterByProfile(profile *Profile, client *rosacli.Client, waitForClu
 
 	return description, err
 }
+
+func WaitForClusterUninstalled(client *rosacli.Client, cluster string, timeoutMin int) error {
+
+	endTime := time.Now().Add(time.Duration(timeoutMin) * time.Minute)
+	sleepTime := 0
+	for time.Now().Before(endTime) {
+		output, err := client.Cluster.DescribeClusterAndReflect(cluster)
+		if err != nil {
+			outputInfo, err := client.Cluster.DescribeCluster(cluster)
+			if strings.Contains(outputInfo.String(), "There is no cluster with identifier or name '"+cluster+"'") {
+				log.Logger.Infof("Cluster %s has been deleted.", cluster)
+				return nil
+			} else {
+				return err
+			}
+		}
+
+		switch output.State {
+		default:
```

Comment:
Remove state check. Just when `state=="uninstalling"` then wait, otherwise return the error message of `fmt.Errorf("Cluster %s is in status of %s which won't be deleted, stop waiting", cluster, state)`